Cargo.toml

```
@@ -249,125 +249,125 @@ sandbox-wasmer = { package = "wasmer", version = "2.2", features = ["singlepass"
-sp-std = { version = "8.0.0", git = "https://github.com/gear-tech/substrate.git", branch = "gear-polkadot-v1.0.0-canary", default-features = false }
-sp-state-machine = { version = "0.28.0", git = "https://github.com/gear-tech/substrate.git", branch = "gear-polkadot-v1.0.0-canary", default-features = false }
-sp-staking = { version = "4.0.0-dev", git = "https://github.com/gear-tech/substrate.git", branch = "gear-polkadot-v1.0.0-canary", default-features = false }
-sp-storage = { version = "13.0.0", git = "https://github.com/gear-tech/substrate.git", branch = "gear-polkadot-v1.0.0-canary", default-features = false }
-sp-timestamp = { version = "4.0.0-dev", git = "https://github.com/gear-tech/substrate.git", branch = "gear-polkadot-v1.0.0-canary", default-features = false }
-sp-transaction-pool = { version = "4.0.0-dev", git = "https://github.com/gear-tech/substrate.git", branch = "gear-polkadot-v1.0.0-canary", default-features = false }
-sp-transaction-storage-proof = { version = "4.0.0-dev", git = "https://github.com/gear-tech/substrate.git", branch = "gear-polkadot-v1.0.0-canary", default-features = false }
-sp-trie = { version = "22.0.0", git = "https://github.com/gear-tech/substrate.git", branch = "gear-polkadot-v1.0.0-canary", default-features = false }
-sp-version = { version = "22.0.0", git = "https://github.com/gear-tech/substrate.git", branch = "gear-polkadot-v1.0.0-canary", default-features = false }
-sp-wasm-interface = { version = "14.0.0", git = "https://github.com/gear-tech/substrate.git", branch = "gear-polkadot-v1.0.0-canary", default-features = false }
-sp-wasm-interface-common = { version = "7.0.0", git = "https://github.com/gear-tech/substrate.git", branch = "gear-polkadot-v1.0.0-canary", default-features = false }
-substrate-build-script-utils = { version = "3.0.0", git = "https://github.com/gear-tech/substrate.git", branch = "gear-polkadot-v1.0.0-canary" }
-substrate-frame-rpc-system = { version = "4.0.0-dev", git = "https://github.com/gear-tech/substrate.git", branch = "gear-polkadot-v1.0.0-canary" }
-substrate-rpc-client = { version = "0.10.0-dev", git = "https://github.com/gear-tech/substrate.git", branch = "gear-polkadot-v1.0.0-canary" }
-substrate-state-trie-migration-rpc = { version = "4.0.0-dev", git = "https://github.com/gear-tech/substrate.git", branch = "gear-polkadot-v1.0.0-canary" }
-substrate-test-client = { version = "2.0.0", git = "https://github.com/gear-tech/substrate.git", branch = "gear-polkadot-v1.0.0-canary" }
-substrate-wasm-builder = { version = "5.0.0-dev", git = "https://github.com/gear-tech/substrate.git", branch = "gear-polkadot-v1.0.0-canary" }
-try-runtime-cli = { version = "0.10.0-dev", git = "https://github.com/gear-tech/substrate.git", branch = "gear-polkadot-v1.0.0-canary" }
+frame-benchmarking = { version = "4.0.0-dev", git = "https://github.com/gear-tech/polkadot-sdk.git", branch = "gear-release-polkadot-v1.1.0", default-features = false }
```

Maybe simplify deps name?
```suggestion
frame-benchmarking = { version = "4.0.0-dev", git = "https://github.com/gear-tech/polkadot-sdk.git", branch = "gear-v1.1.0", default-features = false }
```